.github/workflows/e2e-infra-deploy.yml

```diff
@@ -0,0 +1,70 @@
+# PROCESS
+#
+# 1. Deploy the core stack using AWS CDK.
+# 2. Deploy the AOT stack using AWS CDK.
+
+# USAGE
+#
+# This workflow is used by the E2E Tests workflow to deploy the necessary infrastructure.
+
+name: E2E Tests Infra deployment
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  deploy-core-stack:
+    runs-on: aws-powertools_ubuntu-latest_8-core
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
```

Comment:
As a best practices, please put the sha256 that matches this version. Dependabot will bump it.